tests/test_dependencies.py

```diff
@@ -32,7 +37,57 @@
         ),
     ],
 )
-def test_get_build_backend(build_system, expected_results):
+def test_get_build_backend(
+    build_system: dict[str, list[str]] | dict[str, str | list[str] | None],
+    expected_results: dict[str, typing.Any] | dict[str, str | list[str] | None],
+):
     pyproject_toml = {"build-system": build_system}
     actual = dependencies.get_build_backend(pyproject_toml)
     assert expected_results == actual
+
+
+def test_get_build_system_dependencies(tmp_context: context.WorkContext):
+    fromager_root = pathlib.Path(os.getcwd())
```

Comment:
This assumes that the tests are always executed from the base directory. I prefer to use `__file__` instead of CWD as reference point. First parent is the `tests` directory, grandparent is the root directory. You could make this a module global, too.
```suggestion
    fromager_root = pathlib.Path(__file__).parent.parent
```

Reply:
Done